hdr_html_canvas_element.md

```diff
@@ -266,167 +197,85 @@ The component signals are mapped to red, green and blue tristimulus values accor
 
 The component signals are mapped to red, green and blue tristimulus values according to the Perceptual Quantizer (PQ) system system specified in Rec. ITU-R BT.2100.
 
-### Conversion between color spaces
-
-#### General
-
-There are several places in the HTML specification where a color space conversion is required (e.g, when [drawing images to a
-canvas](https://html.spec.whatwg.org/multipage/canvas.html#colour-spaces-and-colour-correction), [retrieving image data from a
-canvas](https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-getimagedata), among others being added). There exists a
-standard conversion that is applied to all data in these situations, namely, a conversion using relative colorimetric intent.
-
-This operation is defined for [CSS Predefined Color Spaces](https://www.w3.org/TR/css-color-4/#predefined) in the HTML specification
-[here](https://www.w3.org/TR/css-color-4/#predefined-to-lab).
-
-In this section we define this conversion for the new predefined color spaces.
-
-These conversions are expressed using a connection color space with the system colorimetry specified in Rec. ITU-R BT.2100:
-
-* Red primary: `(0.708, 0.292)`
-* Green chromaticity: `(0.170, 0.797)`
-* Blue chromaticity: `(0.131, 0.046)`
-* White chromaticity (D65): `(0.3127, 0.3290)`
-
-_Note:_ The system colorimetry specified in Rec. ITU-R BT.2100 is identical to that specified in Rec. ITU-R BT.2020.
-
-The conversion from color space A to color space B is performed according to the following steps:
-
-* apply the inverse transfer function of color space A
-* convert to the connection space by multiplying by the connection matrix of color space A
-* convert to color space B by multiplying by by the inverse of the connection matrix of color space B
-* apply any linear light scaling to correctly map the level of perceived diffuse white level in to color space B
-* apply the transfer function of color space B
-
-The domain and range of this conversion consist of all real values and the component signals in the connection color space are real numbers.
-
-#### `srgb-linear`
-
-* Transfer function: identity
-
-* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
-
-* Linear light scaling: 1.0
-
-#### `srgb`
+### Color space conversions
 
-* Transfer function: See [srgb-linear](#srgb-linear)
+#### Background
 
-* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
+In general, application should avoid conversions between color spaces and maintain imagery in its original color space: conversions
```

Comment:
But canvas always converts on input from the specified colorspace to that of the canvas.